setup.py

```diff
@@ -1,72 +1,77 @@
 # -*- coding: utf-8 -*-
+"""Installer for the collective.glossary package."""
+
 from setuptools import find_packages
 from setuptools import setup
 
-version = '1.0b2.dev0'
-description = 'A Dexterity-based content type to define a glossary and its terms.'
-long_description = (
-    open('README.rst').read() + '\n' +
-    open('CONTRIBUTORS.rst').read() + '\n' +
-    open('CHANGES.rst').read()
+
+long_description = "\n\n".join(
+    [
+        open("README.rst").read(),
+        open("CONTRIBUTORS.rst").read(),
+        open("CHANGES.rst").read(),
+    ]
 )
 
+
 setup(
-    name='collective.glossary',
-    version=version,
-    description=description,
+    name="collective.glossary",
+    version="1.0a1",
```

Comment:
this is clearly a break change, will probably not work for old Plone, we should think in a better number for the new version

Reply:
an update the docs to reflect it.. for Plone < 5.2 should keep the old version